docs/resource_sets/resource_sets.rst

```diff
@@ -0,0 +1,56 @@
+**********************************
+Resource sets and partial compile
+**********************************
+To support constant scalability resource sets can be used. Without the use of those, each time a resource is added,
+updated or removed, the entire model is recompiled.
+By using resource sets, the resources managed by the inmanta server can be grouped in resource sets.
+This allows to perform a certain action (e.g. dry-run, repair run) only on the resources of a specific resource sets.
```

Comment:
Partial dry-runs and partial repairs runs are something we do not yet support at the moment. As such, I wouldn't mention it.